src/AutoMapper.Collection/AutoMapper.Collection.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Collection Add/Remove/Update support for AutoMapper. AutoMapper.Collection adds EqualityComparison Expressions for TypeMaps to determine if Source and Destination type are equivalent to each other when mapping collections.</Description>
     <VersionPrefix>3.1.4</VersionPrefix>
     <Authors>Tyler Carlson</Authors>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard1.3;net45</TargetFrameworks>
```

Comment:
This is the same change that is done in AutoMapper and will be a breaking change because the `netstandard1.1` is not supported.